tilequeue/command.py

```diff
@@ -949,6 +953,41 @@ def tilequeue_enqueue_tiles_of_interest(cfg, peripherals):
     logger.info('%d tiles of interest processed' % n_toi)
 
 
+def tilequeue_consume_tile_traffic(cfg, peripherals):
+    logger = make_logger(cfg, 'consume_tile_traffic')
+    logger.info('Consuming tile traffic logs ...')
+    logger.info(cfg.tile_traffic_log_path)
+
+    iped_dated_coords = None
```

Comment:
I kinda needed to figure out how to name a tuple that contains ip-address of client, request timestamp, coord

so... 'ed suffix has **with** meaning, couldn't figure out a better name that will be self-explanatory, though it is still confusing I guess. Do you have any suggestions?